src/generic/generic-component-message/generic-component-message.scss

```diff
@@ -41,23 +49,48 @@
   }
 
   .ecl-message__title {
+    @supports (display: grid) {
+      grid-column-start: 1;
+    }
+
     font-weight: bold;
     margin-bottom: map-get($ecl-spacing, 'xs');
   }
 
   .ecl-message__body {
+    @supports (display: grid) {
+      grid-column-start: 1;
+    }
+
     margin: 0;
     padding-left: map-get($ecl-spacing, 'xs');
   }
 
   .ecl-message__dismiss {
-    background: transparent url($ecl-assets-path+'images/close.svg') no-repeat
-      center center;
+    @include ecl-focus-outline-border(
+      $thicknes: 3px,
+      $color: map-get($ecl-colors, 'yellow-100'),
+      $offset: 3px
+    );
+
+    @supports (display: grid) {
```

Comment:
maybe the code would be more readable if `@supports` comes after the first declarations, before `&:hover {`. Was it stylelint telling you it was wrong?

Reply:
stylelint did not complain, I just thought it makes more sense like this

Reply:
You don't write a modifier before the default class, do you? IMHO it should be the same logic here: default first, then "if feature X is supported"